misc/ls-extensions/modules/trigger-service/src/main/java/module-info.java

```diff
@@ -0,0 +1,9 @@
+module io.ballerina.LSExtensions.BallerinaTriggerService {
+    requires io.ballerina.lang;
+    requires com.google.gson;
+    requires io.ballerina.central.client;
+    requires io.ballerina.language.server.commons;
+    requires org.eclipse.lsp4j;
+    requires org.eclipse.lsp4j.jsonrpc;
+    requires io.ballerina.language.server.core;
```

Comment:
What's the usage of the core module here?

Reply:
I'm using `LSClientLogger` inside the `BallerinaTriggerService.java`. Should I prevent using that?

Reply:
Yes, that is not exposed outside at the moment. Instead you can do the following.
`languageClient.logMessage()`. With the context information provided in the service's init method, you can access the language client via the language server instance.

Reply:
Thanks a lot for the suggestion. I changed the logger and removed core module dependencies.